logging/logger_registry.go

```diff
@@ -119,87 +117,40 @@ func (lr *loggerRegistry) updateLoggerRegistry(logConfig []LoggerPatternConfig)
 	return nil
 }
 
-func (lr *loggerRegistry) getRegisteredLoggerNames() []string {
+func (lr *Registry) getRegisteredLoggerNames() []string {
 	lr.mu.RLock()
 	defer lr.mu.RUnlock()
-	registeredNames := make([]string, 0, len(globalLoggerRegistry.loggers))
+	registeredNames := make([]string, 0, len(lr.loggers))
 	for name := range lr.loggers {
 		registeredNames = append(registeredNames, name)
 	}
 	return registeredNames
 }
 
-func (lr *loggerRegistry) registerConfig(logConfig []LoggerPatternConfig) error {
```

Comment:
I didn't see why we'd ever call `updateLoggerRegistry` without setting the `logConfig`. I allowed myself this API change to reduce the number of possible states. Such that I could convince myself we weren't creating problems with weird calls to `updateLoggerRegistry` that intentionally weren't calling `registerConfig`.

Never got to the convincing part. Just seeing this now reminded me to double check.